### ISSUE TITLE
You are missing in the Wiki comparison

### ISSUE THREAD

**Issue by guettli** [django-wiki/django-wiki#1043]
You are missing in the Wiki comparison

https://en.wikipedia.org/wiki/Comparison_of_wiki_software

**Comment by benjaoming** [comment]
Thanks for that idea!!

tldr;

It would be great with more visibility! A slightly sensitive feature of django-wiki is the required Django knowledge: This project (application) seeks advantages from being a basic Django app designed for someone building a Django project with knowledge in that area. That makes it very flexible for Django-fit purposes.

If it was to become a wiki system for people without Django knowledge, there are many risks... thinking of really monstrous projects like Discourse that are building around their users' lack of Rails knowledge. Sometimes people ask questions where the answer is: Read "Getting Started" in Django and you will know the answer. While that could be a great recruitment / advertisement for Django, but it might also leave someone disappointed or with wrong expectations.

With all that in mind, the Wiki article has the section "Target audience" that could well fit a note about this.
